example/test_problem/Hydro/ClusterMerger/clean.sh

```diff
@@ -3,3 +3,5 @@ rm -f Record__Note Record__Timing Record__TimeStep Record__PatchCount Record__Du
       Diag* BaseXYslice* BaseYZslice* BaseXZslice* BaseXline* BaseYline* BaseZline* BaseDiag* \
       PowerSpec_* Particle_* nohup.out Record__Performance Record__TimingMPI_* \
       Record__ParticleCount Record__User Patch_* Record__NCorrUnphy FailedPatchGroup* *.pyc Record__LoadBalance
+
+rm -f Record__Center BH_variable.bin
```

Comment:
Do not remove `BH_variable.bin` since you need this file to restart.

Reply:
@ChunYen-Chen I think we should still remove `BH_variable.bin` as this script is supposed to remove everything generated during runtime, including snapshots `Data_*`.

Reply:
@ChunYen-Chen Having said that, now you use `RESTART` instead of `BH_variable.bin`. So never mind.